README.md

```diff
@@ -122,10 +121,9 @@ static String eWONCertificatePath = "/usr/Certificates";
 ```json
     {
         "ServerConfig":{
-            "URL":"192.168.0.124",
+            "URL":"https://192.168.0.124/piwebapi/omf",
```

Comment:
this is misleading. https://<ip> address does not work.  You don't have to change but <host> would be preferable.

Reply:
Would "https://{YOUR-DOMAIN}/piwebapi/omf" be okay?